enterprise/migrations/0106_move_branding_config_colors.py

```diff
@@ -0,0 +1,37 @@
+# Generated by Django 2.2.14 on 2020-08-03 17:24
+
+from django.db import migrations
+from django.db.models import Q
+
+
+def copy_primary_secondary_colors(apps, schema_editor):
+    EnterpriseCustomerBrandingConfiguration = apps.get_model('enterprise', 'EnterpriseCustomerBrandingConfiguration')
+    for branding_config in EnterpriseCustomerBrandingConfiguration.objects.filter(
+        Q(banner_background_color__isnull=False) | Q(banner_border_color__isnull=False)
+    ):
+        print(branding_config)
+        branding_config.primary_color = branding_config.banner_background_color
+        branding_config.secondary_color = branding_config.banner_border_color
+        branding_config.save()
+
+
+def clear_primary_secondary_colors(apps, schema_editor):
+    EnterpriseCustomerBrandingConfiguration = apps.get_model('enterprise', 'EnterpriseCustomerBrandingConfiguration')
+    for branding_config in EnterpriseCustomerBrandingConfiguration.objects.filter(
+        Q(primary_color__isnull=False) | Q(secondary_color__isnull=False)
+    ):
+        branding_config.primary_color = None
+        branding_config.secondary_color = None
+        branding_config.save()
```

Comment:
Do we even need to have a reverse here? I assume it's fine if the old data stayed in the former fields if someone wanted to run back these migrations

Reply:
It would be fine, but it was easy to write and I liked having a way to repeatedly test locally and see the same result.